README.md

```diff
@@ -104,6 +104,17 @@ will be populated with these extras:
 You can register a receiver to receive this intent. An example of such a use is below:
 
 ```java
+public class SampleApplication extends Application {
+  @Override
+  public void onCreate() {
+    super.onCreate();
+    IntentFilter intentFilter = new IntentFilter("com.airbnb.deeplinkdispatch.DEEPLINK_ACTION");
+    LocalBroadcastManager.getInstance(this).registerReceiver(new DeepLinkReceiver(), intentFilter);
+  }
+}
+```
```

Comment:
We usually don't update the readme until the code is included in a final release, otherwise it would be confusing for people using the current release version since the readme won't match it. 
Once we release the next version of the library then we go ahead and update it.